packages/react-ui-ag/src/Carousels/ListingCarousel.js

```diff
@@ -43,8 +44,11 @@ export default class ListingCarousel extends Component {
 
   @autobind
   listingClickHandler(index) {
+    const { onIndexChange, listings } = this.props
+
     if (index !== this.state.selectedIndex) {
       this.updatedSelectedIndex(index)
+      if (onIndexChange) onIndexChange(listings[index])
```

Comment:
In the carousel component in core this is called ‘onSlide’. Can we rename this for consistency?

Reply:
Yeah no problem. I like that better actually.